WooCommerce/StoreWidgets/StoreInfoProvider.swift

```diff
@@ -129,14 +133,41 @@ private extension StoreInfoProvider {
               let storeID = UserDefaults.group?[.defaultStoreID] as? Int64,
               let storeName = UserDefaults.group?[.defaultStoreName] as? String,
               let storeTimeZoneGMTOffset = UserDefaults.group?[.defaultStoreTimeZoneGMTOffset] as? Int,
-              let storeTimeZone = TimeZone(secondsFromGMT: storeTimeZoneGMTOffset) else {
+              let storeTimeZone = TimeZone(secondsFromGMT: storeTimeZoneGMTOffset),
+              let storeCurrencySettingsData = UserDefaults.group?[.defaultStoreCurrencySettings] as? Data,
+              let storeCurrencySettings = try? JSONDecoder().decode(CurrencySettings.self, from: storeCurrencySettingsData) else {
             return nil
         }
-        return Dependencies(authToken: authToken, storeID: storeID, storeName: storeName, storeTimeZone: storeTimeZone)
+        return Dependencies(authToken: authToken,
+                            storeID: storeID,
+                            storeName: storeName,
+                            storeTimeZone: storeTimeZone,
+                            storeCurrencySettings: storeCurrencySettings)
     }
 }
 
 private extension StoreInfoProvider {
+
+    func formattedAmountString(for amountValue: Decimal, with currencySettings: CurrencySettings?) -> String {
+        let currencyFormatter = CurrencyFormatter(currencySettings: currencySettings ?? CurrencySettings())
+        return currencyFormatter.formatAmount(amountValue) ?? Constants.valuePlaceholderText
+    }
+
+    func formattedConversionString(for conversionRate: Double) -> String {
+        let numberFormatter = NumberFormatter()
+        numberFormatter.numberStyle = .percent
+        numberFormatter.minimumFractionDigits = 1
+
+        // do not add 0 fraction digit if the percentage is round
+        let minimumFractionDigits = floor(conversionRate * 100.0) == conversionRate * 100.0 ? 0 : 1
+        numberFormatter.minimumFractionDigits = minimumFractionDigits
+        return numberFormatter.string(from: conversionRate as NSNumber) ?? Constants.valuePlaceholderText
```

Comment:
I think there is a bit of an edge case here or in the conversion formula as because `100%` is showing as `10.000%`
![IMG_718F09EAB50E-1](https://user-images.githubusercontent.com/562080/189761757-c826f814-ebd1-48e9-97d6-2e7ff1d3d1ff.jpeg)

In retrospective, I should have added unit tests to these types 🙈

Reply:
Yeah, very obvious error, converted rate value to percentage twice 😄
Fixed in 72e118c.